src/eko/evolution_operator/__init__.py

```diff
@@ -144,42 +160,32 @@ def quad_ker(
             number of evolution steps
         ev_op_max_order : int
             perturbative expansion order of U
+        is_sv_scheme_a : bool
+            use scale variation scheme A
 
     Returns
     -------
         ker : float
             evaluated integration kernel
     """
-    is_singlet = mode[0] == "S"
-    # get transformation to N integral
-    if logx == 0.0:
-        return 0.0
-    r = 0.4 * 16.0 / (-logx)
-    if is_singlet:
-        o = 1.0
-    else:
-        o = 0.0
-    n = mellin.Talbot_path(u, r, o)
-    jac = mellin.Talbot_jac(u, r, o)
-    # check PDF is active
-    if is_log:
-        pj = interpolation.log_evaluate_Nx(n, logx, areas)
-    else:
-        pj = interpolation.evaluate_Nx(n, logx, areas)
-    if pj == 0.0:
+    ker_base = QuadKerBase(u, is_log, logx, mode)
+    integrand = ker_base.integrand(areas)
+    if integrand == 0.0:
         return 0.0
+
     # compute the actual evolution kernel
-    if is_singlet:
-        gamma_singlet = gamma_singlet_fact(order, n, nf, L)
+    if ker_base.is_singlet:
+        gamma_singlet = ad.gamma_singlet(order, ker_base.n, nf)
+        if is_sv_scheme_a:
```

Comment:
Actually, do we need this guard? or can't we rely on L=0? if ever, add an early exit in `gamma_fact`?